tests/unit/sagemaker/test_deserializers.py

```diff
@@ -0,0 +1,34 @@
+# Copyright 2018-2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+from __future__ import absolute_import
+
+import pytest
+
+from sagemaker import deserializers
+
+
+class BaseDeserializerTest:
+    def test_instantiate_deserializer_without_accept_attribute(self):
```

Comment:
you're testing whether `ABCMeta` functionality works -- standard library functionality/core cpython implementation details. you don't need to test this. it's being tested at the language level.

`@abc.abstractmethod` sets the `__isabstractmethod__` [on the func object](https://github.com/python/cpython/blob/5c117dd227e1b4c4f0a62564d8592f1ba45c91eb/Lib/abc.py#L24) and [enforcement of override](https://github.com/python/cpython/blob/5c117dd227e1b4c4f0a62564d8592f1ba45c91eb/Lib/abc.py#L10-L12) is during instantiation

Reply:
Fixed